draft-ietf-rtcweb-jsep.xml

```diff
@@ -1675,15 +1666,50 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               policy is set to "balanced", and this is not the first m= section
               for this media type, an "a=bundle-only" line.</t>
 
-              <t>If the RtpTransceiver's RtpSender is active, an "a=rid" line for each
-              encoding specified by the application in the RtpSenders's parameters.
-              If no encodings have been specified, then a single "a=rid" line is generated.
-              The "a=rid" line is specified in <xref target="I-D.ietf-mmusic-rid"></xref>, and its direction MUST be "send".  If the application has chosen a RID
+              <t>If the RtpSender of the RtpTransceiver associated with this
+              m=section is active:
+                <list style="symbols">
+                  <t>An "a=msid" line, as specified in <xref target=
```

Comment:
I don't think you can change the MSID unless you recycle the MID, but this section is for initial offers.  Not changing the MSID is down below.